File: examples/quran/summary/node/getChapterSummary.js
const { QuranService } = require("m3o/quran");

// Get a summary for a given chapter (surah)
async function getChapterSummary() {
  let quranService = new QuranService(process.env.MICRO_API_TOKEN);
  let rsp = await quranService.summary({
    chapter: 1,
  });
  console.log(rsp);
}

getChapterSummary();

File: examples/quran/summary/go/getChapterSummary.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/quran"
)

// Get a summary for a given chapter (surah)
func GetChapterSummary() {
	quranService := quran.NewQuranService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := quranService.Summary(&quran.SummaryRequest{
		Chapter: 1,
	})
	fmt.Println(rsp, err)
}
